sdk/network/mgmt/src/main/java/com/azure/management/network/implementation/LoadBalancerImpl.java

```diff
@@ -79,7 +79,6 @@
 
     @Override
     protected Mono<LoadBalancerInner> getInnerAsync() {
-        // TODO(not known): Extra parameter
```

Comment:
Eh, I mean, you could try delete the `null` in `.getByResourceGroupAsync(this.resourceGroupName(), this.name(), null);`. If it compiles then all is good. If not let me know. It should works same on ``.getByResourceGroupAsync(this.resourceGroupName(), this.name());`

Reply:
Done